app/scripts/directives/events.js

```diff
@@ -31,10 +31,24 @@ angular.module('openshiftConsole')
         };
 
         var sortedEvents = [];
+        var currentID = _.get($scope, 'sortConfig.currentField.id');
         var sortEvents = function() {
-          // TODO: currentField is renamed in angular-patternfly 3.0
-          var sortID = _.get($scope, 'sortConfig.currentField.id', 'lastTimestamp'),
-              order = $scope.sortConfig.isAscending ? 'asc' : 'desc';
+          var sortID = _.get($scope, 'sortConfig.currentField.id', 'lastTimestamp');
+          // only change if sort field is changed
+          if (currentID !== sortID) {
+            // set currentID to sortID
+            currentID = sortID;
+            // if currentID is lastTimestamp and isAscending is true, set
+            // isAscending to false so records display reverse sorted by time
+            if (currentID === 'lastTimestamp' && $scope.sortConfig.isAscending === true) {
```

Comment:
i would take this approach

```
var defaultIsReversed = {
  lastTimestamp: true
};
if (current ID !== sortID) {
  $scope.sortConfig.isAscending = !(defaultIsReversed[currentID]);
}
```

That lets you add more reversed sorts easily in the future without having to check them all, and lets you do the assignment in one single line without the nested if/else block